portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/RemoteApps.testcase

```diff
@@ -49,19 +49,34 @@ definition {
 		property portal.acceptance = "true";
 
 		task ("Create Vanilla Counter as a Custom Element") {
-			JSONRemoteApp.addCustomElementRemoteAppEntry(
-				customElementHTMLName = "vanilla-counter",
-				customElementName = "Vanilla Counter",
-				customElementURL = "http://remote-component-test.wincent.com/packages/vanilla-counter/index.js");
+			RemoteApps.goToRemoteAppsPortlet();
+
+			RemoteApps.addCustomElementAllFields(
+				entryCSSURL = "http://remote-component-test.wincent.com/index.css",
```

Comment:
We shouldn't have any references to the `wincent.com` domain

Reply:
agree, we'll prioritize https://issues.liferay.com/browse/LPS-142823 to get this updated.